src/components/AppAlertGroup/index.tsx

```diff
@@ -47,17 +47,17 @@ class AppAlertGroup extends React.PureComponent<Props, State> {
   }
 
   private getAlert(item: AlertItem): React.ReactElement {
-    const { variant, title, key } = item;
+    const { variant, title, key, children } = item;
     const showAlertTimer = setTimeout(() => {
       this.appAlerts.removeAlert(key);
-    }, variant === AlertVariant.success ? 2000 : 8000);
+    }, variant === AlertVariant.success ? 2000 : variant !== AlertVariant.info ? 8000 : 20000);
```

Comment:
This line became hard to read. Please move calculating the timeout delay above `setTimeout`.

Reply:
Ok. I have changed it.